### PR TITLE
create component and container for last channels message list

### LINKED ISSUE #4: Render a list of recent conversations
## User story

**Who:** **As a** user

**What:** **I want to** See the users i latest chatted with

**Why:** **so that we can** quickly chat with them again

## Implementation details
- What if there are no latest users? Maybe a "create your first conversation" button
- Limit the number of conversations to 20
- See figure 1 for ui
- You need to show: profile image, name of person, the latest message (one line) and time of message
- If there is no api, make your own super simple api that just returns 3 users with image, name, etc. Talk to the person building the api. How will the data be structured? You want to have the same structure in your test api as the correct one

nice to have as extra PR: automatic load new conversations when scrolling down

### PR DESCRIPTION
**Pull request by afrouzhakim**
# Description
I made the component and container to show the list of last channels' message. 

Fixes #4 

# How to test.
You can run "npm run storybook" 

# Checklist
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/LastMessageList/LastMessagesList.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { object } from '@storybook/addon-knobs';
+import './LastMessagesList.css';
+
+export default function LastMessageList({ lastChannels }) {
+  if (!lastChannels) {
+    return <p>No channels, start a conversation to see it here</p>;
+  }
+  return (
+    <ul>
+      {lastChannels.map(({ message, updated_at, fk_channel_id, title }) => (
```
Comment (tpetrina) [inline src/client/components/LastMessageList/LastMessagesList.js:13]: I would prefer if you renamed these in the controller to a nice camelCase alternative and then use those in frontend.

**Review by tpetrina — COMMENTED** [review]
Remove changes in `LastMessagesList.css` file.

File: src/client/components/LastMessageList/LastMessagesList.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { object } from '@storybook/addon-knobs';
```
Comment (benna100) [inline src/client/components/LastMessageList/LastMessagesList.js:4]: I dont think you should add the story book knobs in the normal component. Pretty sure that is only for the storybook component. Or am i wrong?

**Review by benna100 — CHANGES_REQUESTED** [review]
This makes me happy to see. Great work Afrouz! 

Especially with the controller and containers distinction. Very impressive stuff :) There are a few comments to fix. 

Also please add this to the Overview page

File: src/client/components/LastMessageList/LastMessagesList.js (hunk 2 above)
Comment (afrouzhakim) [reply src/client/components/LastMessageList/LastMessagesList.js:4]: I remove that but it was just in lastMessagesList

File: src/client/components/LastMessageList/LastMessagesList.js (hunk 1 above)
Comment (afrouzhakim) [reply src/client/components/LastMessageList/LastMessagesList.js]: it is done.

**Review by benna100 — APPROVED** [review]

**Pull request merged**